### PR TITLE
[Pages] Add Crisp React instructions to deployment guides

### PR DESCRIPTION
**Pull request by winwiz1**

### CODE REVIEW

File: products/pages/src/content/framework-guides/deploy-a-crisp-react-application.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import TutorialsBeforeYouStart from "../_partials/_tutorials-before-you-start.md" 
+
+# Deploy a Crisp React app
+
+[Crisp React](https://github.com/winwiz1/crisp-react) is an open-source boilerplate for building modern web applications. It has small codebase and a simple build with focus on performance and maintainability while offering build-time SSR and ability to split a monolithic React app into multiple SPAs.
+In this guide, you will create a new Crisp React application and deploy it using Cloudflare Pages.
+
+## Setting up a new project
+
+Create a new project by cloning the Crisp React repository:
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/deploy-a-crisp-react-application.md:14]:
```suggestion
Create a new project by running the following commands in your terminal to clone the Crisp React repository:
```

File: products/pages/src/content/framework-guides/deploy-a-crisp-react-application.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+pcx-content-type: how-to
+---
+
+import TutorialsBeforeYouStart from "../_partials/_tutorials-before-you-start.md" 
+
+# Deploy a Crisp React app
+
+[Crisp React](https://github.com/winwiz1/crisp-react) is an open-source boilerplate for building modern web applications. It has small codebase and a simple build with focus on performance and maintainability while offering build-time SSR and ability to split a monolithic React app into multiple SPAs.
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/deploy-a-crisp-react-application.md:9]:
```suggestion
[Crisp React](https://github.com/winwiz1/crisp-react) is an open-source boilerplate for building modern web applications. It has a small codebase and a simple build with a focus on performance and maintainability while offering build-time server-side rendering (SSR) and the ability to split a monolithic React app into multiple single-page applications (SPAs).
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]
@signalnerve and @lukeed can you please look this over?

**Comment by lukeed** [comment]
Hey @winwiz1, congratulations on your project – that's awesome! And thank you for your PR, but at this time we'll unfortunately have to close it. Our list of framework guides is meant to target popular/well-known community choices so that it's not overwhelmingly showing everything. 

I'd like to revisit this submission in the future once Crisp's user base has had some time to grow, though :) Good luck!

**Pull request closed without merging**